From 3af68d4ae3962442d8bae475925d92be8aea5b6d Mon Sep 17 00:00:00 2001
From: Sunil V L <sunilvl@ventanamicro.com>
Date: Thu, 3 Mar 2022 19:15:16 +0530
Subject: [RFC V1 5/5] uefi-sct/SctPkg: Add a category for
 RISCV_EFI_BOOT_PROTOCOL

REF: https://bugzilla.tianocore.org/show_bug.cgi?id=3837

This patch provides a name for the GUID of RISCV_EFI_BOOT_PROTOCOL
in the SCT.

Signed-off-by: Sunil V L <sunilvl@ventanamicro.com>
---
 uefi-sct/SctPkg/Config/Data/Category.ini | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/uefi-sct/SctPkg/Config/Data/Category.ini b/uefi-sct/SctPkg/Config/Data/Category.ini
index af27e362..2db2befd 100644
--- a/uefi-sct/SctPkg/Config/Data/Category.ini
+++ b/uefi-sct/SctPkg/Config/Data/Category.ini
@@ -1026,3 +1026,10 @@ InterfaceGuid = 8D59D32B-C655-4AE9-9B15-F25904992A43
 Name          = IHV\ConsoleSupportTest\AbsolutePointerProtocolTest
 Description   = Absolute Pointer Protocol Test on IHV Drivers
 
+[Category Data]
+Revision      = 0x00010000
+CategoryGuid  = CCD15FEC-6F73-4EEC-8395-3E69E4B940BF
+InterfaceGuid = CCD15FEC-6F73-4EEC-8395-3E69E4B940BF
+Name          = RiscvBootProtocolTest
+Description   = RISC-V Boot Protocol Test
+
-- 
2.25.1

